packages/apidom-ls/src/config/openapi/security-scheme/documentation.ts

```diff
@@ -1,42 +1,41 @@
 const documentation = [
   {
```

Comment:
The changes in this file make this docs rules consistent with AsyncAPI docs rules.